styles.css

```diff
@@ -31,19 +20,35 @@
 }
 
 .menu {
-  width: 18px;
-  height: 12px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
   padding: 8px;
-  border-radius: 8px;
-  margin-right: 10px;
+  height: 48px;
+  align-items: center;
+}
+
+.menu a {
+  text-decoration: none;
 }
 
 .headlines {
-  padding-top: 12px;
   background: #1c1a19;
-  min-height: calc(100vh - 81px);
+  min-height: calc(100vh - 70px);
   display: flex;
-  flex-direction: column;
+  align-items: center;
+  background-image: url('images/back.png');
+  background-repeat: no-repeat;
```

Comment:
- Hey, great job addressing the issues raised by the previous reviewer, however, your design is not having the background image. Kindly check why the image is not been applied to your page.

*your design*
![Screenshot from 2023-01-14 01-22-30](https://user-images.githubusercontent.com/59397159/212440844-d63dd58f-438c-43e2-b6ba-fe47dd695052.png)

*template design*
![Screenshot from 2023-01-14 01-23-08](https://user-images.githubusercontent.com/59397159/212440837-34bf4dc9-91f6-4278-8680-52a44c680fea.png)

Reply:
Hi ziecco, 
Thank you for reviewing my codes so i am going to solve it